core/micro-services/build.sbt

```diff
@@ -1,8 +1,9 @@
 lazy val WorkflowCore = project in file("workflow-core")
+lazy val Dao = project in file("dao")
```

Comment:
name it DAO in code. it is an abbreviation for `Data Access Object`. you can keep `dao` in directory or package name.

Reply:
renamed